samples/GraphQL.DataLoader.Example/GraphQl/SalespeopleByNameDataLoader.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DataLoaderGql.Types;
+using GraphQL.DataLoader;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataLoaderGql.GraphQl;
+
+public class SalespeopleByNameDataLoader(DealershipDbContext db) : DataLoaderBase<string, Salesperson>
```

Comment:
this example promotes the use of data loaders registered as services.  While this technique is documented, and in fact I use it often in my own schemas, we don't promote this as the primary way to make data loaders.  What's your take on this?  Perhaps we should demonstrate two different methods of making data loaders here within this sample?  One using the `IDataLoaderContextAccessor` (typical) and also via DI?  My foremost thought is that incorrect configuration of DI-based data loaders can lead to memory leaks.  It also is slightly more complex to set up, and may require a serial execution strategy, which is not often the typical use case.  Your thoughts?

Reply:
I think one of each is a good idea.
I dont think these Risks of DI dataloader jump out of the docs tho. Should probably added in scope of the Dataloader Docs Bounty Issue